### PR TITLE
Add Ubuntu 20.04 support

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
needs to wait for a bolt release...

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,10 @@ matrix:
     bundler_args: --without development release
     env: PUPPET_INSTALL_TYPE=agent BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_debug=true BEAKER_setfile=ubuntu1804-64m BEAKER_HYPERVISOR=docker CHECK=beaker
     services: docker
```
Comment (dhoppe) [inline .travis.yml:40]: The task for Puppet 5 is missing.

Comment (mmoll) [reply .travis.yml:40]: There's no puppet-agent 5.x for focal and I'm unsure if it will be released, but let's wait and see. :)

Comment (ekohl) [reply .travis.yml:40]: AFAIK it wasn't for Debian 10 so I doubt we'll see it for Ubuntu 20.04

**Comment by ekohl** [comment]
Looks like there is no puppet-bolt package for Ubuntu 20.04. Should we skip that for now?

**Comment by mmoll** [comment]
I'd wait for the release as I need the integration tests to be sure, it works.

File: .travis.yml (hunk 1 above)
Comment (dhoppe) [reply .travis.yml:40]: There will be no packages for Puppet 5:
* https://tickets.puppetlabs.com/browse/CPR-737

**Comment by mmoll** [comment]
Seems like this is working OOTB. :green_heart:

**Comment by ekohl** [comment]
@dhoppe you requested changes, but I think those are no longer relevant now? Mind merging?

**Comment by dhoppe** [comment]
@ekohl My request was related to Puppet 5, but we all figured out that Puppet Inc. will not provide any Puppet 5 packages for Ubuntu 20.04.

**Pull request merged**